app/src/main/java/net/sourceforge/opencamera/sensorlogging/VideoPhaseInfo.java

```diff
@@ -0,0 +1,48 @@
+package net.sourceforge.opencamera.sensorlogging;
+
+import androidx.annotation.NonNull;
+
+import java.util.List;
+
+public class VideoPhaseInfo {
+    private final long mVideoPhaseNs;
+    private final double mAvgDurationNs;
+
+    // Accepts a sorted collection
+    private static double median(List<Long> numList) {
```

Comment:
If this function is actually useful now and the performance is not critical, just call `Collections.sort`. It is safe to be called on small sorted List, but will guarantee the sorted order.